independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,569 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HubSpot CMS",
+    "description": "HubSpot CMS community connector.",

```

Comment:
@troystaylor Can you please provide valid description as it should describe concisely the main purpose and value offered by your connector.

Documentation - [Description of connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)